isaric/parsers/ccp-ghana.toml

```diff
@@ -1332,6 +1336,12 @@
   date = { field = "icu_date" }
   value = { field = "ph", description = "pH" }
 
+[[observation]]
+  name = "fio2_percent"
+  phase = "study"
+  date = { field = "vitals_date" }
+  value = { field = "fi02", apply = { function = "Percentage" } }
```

Comment:
@pipliggins just to check, the function "Percentage" converts the fractional values (0.21-1.0) to percentage (21-100%)?

Reply:
Yes! It will also leave any existing percentages alone, as many datasets seem to have a mix of both regardless of the supposed unit.